src/state/queries/feed.ts

```diff
@@ -216,10 +227,11 @@ const FOLLOWING_FEED_STUB: FeedSourceInfo = {
   likeCount: 0,
   likeUri: '',
 }
-const DISCOVER_FEED_STUB: FeedSourceInfo = {
+const PWI_DISCOVER_FEED_STUB: FeedSourceInfo = {
+  isPrimaryAlgorithm: true,
   type: 'feed',
   displayName: 'Discover',
-  uri: '',
+  uri: DISCOVER_FEED_URI,
```

Comment:
Another example of feeds driven by URI. And in the case of PWI, Discover is for all intents and purposes, the primary algo.